test/e2e/azure_test.go

```diff
@@ -234,4 +234,53 @@ var _ = Describe("Workload cluster creation", func() {
 			})
 		})
 	})
+
+	// ci-e2e.sh and Prow CI skip this test by default, since N-series GPUs are relatively expensive
```

Comment:
when running this locally, will ci-e2e.sh skip the test? I don't see the `GINKGO_SKIP` default being set anywhere

Reply:
nevermind, it's in the Makefile